### PR TITLE
Remove _libcall usage

### PR DESCRIPTION
**Pull request by gramalingam**
Remove the explicit usage of _libcall. It's not needed.

### CODE REVIEW

File: onnxscript/converter.py [hunk 1]
```diff
@@ -844,21 +844,16 @@ def translate_opset_expr(self, node) -> values.Opset:
 
     def translate_callee_expr(self, node) -> values.Op:
         """Return an Op"""
```
Comment (justinchuby) [inline onnxscript/converter.py:846]: Worth fixing this check?

**Review by justinchuby — APPROVED** [review]
Probably not the best person to review but lgtm

File: onnxscript/converter.py (hunk 1 above)
Comment (gramalingam) [reply onnxscript/converter.py:847]: Done (I hope). I added the `NoReturn` annotation to the `fail` methods.

Comment (justinchuby) [reply onnxscript/converter.py:847]: Thanks!

Comment (justinchuby) [reply onnxscript/converter.py:847]: Looks like CodeQL isn't smart enough. that's ok

**Pull request merged**